helixlauncher-core/src/meta.rs

```diff
@@ -0,0 +1,91 @@
+use std::io;
+
+use reqwest::Client;
+use thiserror::Error;
+
+use crate::config::Config;
+
+pub struct HelixLauncherMeta<'a> {
+    client: Client,
+    config: &'a Config,
+}
+
+impl<'a> HelixLauncherMeta<'a> {
+    pub fn new(config: &'a Config) -> HelixLauncherMeta<'a> {
+        HelixLauncherMeta {
+            client: Client::new(),
```

Comment:
Creating a new Client instance on each invocation is bad, though if the Meta instance is kept around that would probably be fine

Reply:
Well alternatively the client would need to be kept around (or newly instanciated each time) so this shouldnt make much of a difference